pyunifiprotect/unifi_protect_server.py

```diff
@@ -329,11 +338,24 @@ async def _setup_websocket(self):
                         _LOGGER.exception("Error processing websocket message")
                         return
                 elif msg.type == aiohttp.WSMsgType.ERROR:
+                    _LOGGER.exception("Error from Websocket: %s", msg.data)
                     break
         finally:
             _LOGGER.debug("websocket disconnected")
             self.ws_connection = None
 
+    def _log_websocket_failure(self):
+        now = datetime.now()
+        if self._websocket_error_start is None:
+            self._websocket_error_start = now
+
+        self._websocket_failures += 1
+        log = _LOGGER.warning
+
+        if self._websocket_failures < 10 or (now - self._websocket_error_start) < WEBSOCKET_ERROR_GRACE_PERIOD:
+            log = _LOGGER.info
```

Comment:
```suggestion
            log = _LOGGER.debug
```

A lot of users have their logging set at info and will still generate issues from this.  I think we should leave this at debug